seq/detail/window.cpp

```diff
@@ -109,9 +109,10 @@ bool Window::configExitGL()
 bool Window::processEvent( const eq::Event& event )
 {
     seq::Renderer* const renderer = getRenderer();
-    if( renderer->processEvent( event ))
-        return true;
-    return eq::Window::processEvent( event );
+    
+    bool successProcess = renderer->processEvent( event );
+
+    return eq::Window::processEvent( event ) || successProcess;
```

Comment:
Yeah, I am not sure if this is hiding the bug actually but I thought that eq::Window has to handle the event also. You can reproduce the behavior I am describing with seqPly like this:

```c++
virtual bool Renderer::processEvent( const eq::Event& event )
{
    switch( event.type )
    {   
    case eq::Event::WINDOW_POINTER_BUTTON_PRESS:
    {   
        return true;
    }   
    default:
    {   
        return false;
    }   
    }  
}

```